src/internal/client/pfs.go

```diff
@@ -41,11 +41,13 @@ func NewBranch(projectName, repoName, branchName string) *pfs.Branch {
 
 // NewCommit creates a pfs.Commit in the given project, repo & branch.
 func NewCommit(projectName, repoName, branchName, commitID string) *pfs.Commit {
-	return &pfs.Commit{
-		Repo:   NewRepo(projectName, repoName),
-		Id:     commitID,
-		Branch: NewBranch(projectName, repoName, branchName),
+	commit := &pfs.Commit{
```

Comment:
What does this change?

Reply:
Ahh, history of debugging. Let me clean that..